app/views/assessments/show.html.erb

```diff
@@ -156,7 +155,17 @@
           <% end %>
         <% end %>
       <% end %>
+      <% if @cud.instructor? then %>
+        <li class="attachment add">
+          <%= link_to new_course_assessment_attachment_path(@course, @assessment) do %>
+            <span class="glyphicon glyphicon-plus"></span><%= "Add #{@assessment.display_name} Attachment" %>
+          <% end %>
+        </li>
+      <% end %>
```

Comment:
We need to move this logic outside of the outer `if` block to expose the link even when there aren't attachments.